kafka-connect-jms/src/test/scala/com/datamountaineer/streamreactor/connect/sink/writer/JMSWriterTest.scala

```diff
@@ -105,7 +105,13 @@ class JMSWriterTest extends TestBase with Using with BeforeAndAfter with Convert
 
         val kcqlQ= getKCQL(queueName, kafkaTopic1, "QUEUE")
         val kcqlT= getKCQL(topicName, kafkaTopic2, "TOPIC")
-        val props = getSinkProps(s"$kcqlQ;$kcqlT", s"$kafkaTopic1,$kafkaTopic2", brokerUrl)
+        val messageType = "TextMessage"
```

Comment:
I'm not sure what's the standard here, but shouldn't message type be full package+class-name ?...

Reply:
JMS message type is just a header. In our spec we should use `TextMessage`, so that's why I tested the same value here.